examples/rotator_test_light.py

```diff
@@ -0,0 +1,95 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+import logging
+import yaml
+
+import numpy as np
+from matplotlib import pyplot as plt
+import astropy.units as u
+from astropy.time import Time
+from astropy.coordinates import AltAz, ICRS, EarthLocation, Angle, FK5
+import asyncio
+from lsst.ts import salobj
+from lsst.ts.idl.enums import ATPtg
+
+from astropy.utils import iers
+iers.conf.auto_download = False
+
+
+async def amain():
+
+    d = salobj.Domain()
+
+    mtptg = salobj.Remote(d, "MTPtg")
+
+    await mtptg.start_task
+
+    await salobj.set_summary_state(mtptg, salobj.State.ENABLED)
+
+    location = EarthLocation.from_geodetic(lon=-70.747698*u.deg,
```

Comment:
i suppose I can't yell about how you should be getting this from the pointing component since it's not publishing it. Maybe add a comment saying that this will get replaced one day?

Reply:
done